### ISSUE TITLE
MSX Sound Enhancer

### ISSUE THREAD

**Issue by ReNardier** [openMSX/openMSX#1787]
https://www.msx.org/wiki/Lost_Technology_MSX_Sound_Enhancer
https://www.youtube.com/watch?v=KrZWmedqcfE
before you ask(and although im again out of my depth), this one seems trickier than the beepack due to the thing it accepts being a sound cartridge(although perhaps something funky & quite likely unintended could be accomplished with another type of cart), so not only does this one require that it can specify which cart slot it occupies, but the proper cart-based extension should be selectable to be on top of it in the same daisy-chaining manner as the bee cards do on the beepack, sorry again if this(i) is(was) awfully unhelpful, and never-the-less have a good day

**Comment by m9710797** [comment]
I'm not familiar with this extension, but if I read the documentation correctly, this is nothing more than the combination of MSX-Music (OPLL) (with optional ROM) and PSG (YMZ294 or AY8910). The documentation also talks about a non-standard balance between these two chips between different MSX machines. But that balance can already be controlled in openMSX.

Maybe I'm missing something. But at the moment I don't see a reason why we should emulate this in openMSX. I mean it's extra effort, but it doesn't bring much (any?) value.

But of course feel free to comment if I'm mistaken. I'll keep this ticket open for a little while longer.

**Comment by MBilderbeek** [comment]
@ReNardier can you explain the value of having this emulated?

**Comment by madscient** [comment]
From author of that device,
This device only improves the quality of the audio signal output from the real machine and does not add any software controllable features.
The only features that can be achieved by the emulator are volume balance adjustment and pseudo-stereo, but the former has already been implemented, and the latter is a matter of taste. If you need pseudo-stereo, you can simply use a combination of existing generic sound plug-ins.
I don't see the point of having emulated as “MSX Sound Enhancer”.

**Comment by m9710797** [comment]
Ok, thanks for the info.
Let's close this ticket.

**Comment by MBilderbeek** [comment]
Yes, thanks a lot for reaching out @madscient ! Much appreciated.
